packages/sourcecred/src/api/instance/localInstance.js

```diff
@@ -200,6 +206,23 @@ export class LocalInstance implements Instance {
     ).then((result) => Ledger.parse(result));
   }
 
+  async writeGrainIntegrationOutput(
+    output: GrainIntegrationOutput,
+    distribution: Distribution
+  ): Promise<void> {
+    mkdirx(pathJoin(...GRAIN_INTEGRATION_DIRECTORY));
+    const credDateString = new Date(
+      distribution.credTimestamp
+      // utilise the `SE` datestring format since it appears like `YYYY-MM-DD`
+      // with numbers and is therefore easy to sort on and human-readable.
+    ).toLocaleDateString("en-SE");
```

Comment:
we have a util/timestamp.js module that has a toISO function that does this. Probably better to use it.